src/components/Footer/Archives.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Archives = () => (
+  <div className="col-two md-four mob-full s-footer__archives">
+    <h4>Archives</h4>
+    <ul className="s-footer__linklist">
+      <li><a href="#0">July 2019</a></li>
```

Comment:
Also here, a loop would be better IMO.

Reply:
```suggestion
      const list = () => {
          let months = [ "January", "February", "March", "April", "May", "June",
          "July", "August", "September", "October", "November", "December" ];
          let newDate = new Date();
          let dates = [];
          for (let i = 0; i < 6; i += 1) {
              dates.push(<li><a href="#0">{months[newDate.getMonth() - i]} {newDate.getFullYear()} </a></li>)
        }
        return dates;
  };
```

I like this one better! :)